setup.py

```diff
@@ -0,0 +1,40 @@
+from setuptools import setup
+
+with open('README.md', 'r') as fh:
+    long_description = fh.read()
```

Comment:
Just a heads up, I had this fail once on install (could not find the README file), but I think this was an environment issue as I repackaged and it went away. I have had some strange interactions with pip and pyenv, but they are generally transient and go away with a clean install.

Reply:
Good to know to look out for. This code runs when building the package, not installing it, right?

Reply:
This error happened to me when I was using it client side, as if the package was trying to read that file after installation, instead of during installation or even packaging, which would make more sense to me. Now I wish I had triaged more beyond reinstalling everything (which fixed it), so I'm not sure what the actual constellation of states was that brought it about.